src/pages/RepoPage/CoverageOnboarding/ActivationBanner/TrialEligibleBanner/TrialEligibleBanner.spec.tsx

```diff
@@ -58,6 +58,19 @@ describe('TrialEligibleBanner', () => {
     expect(list).toBeInTheDocument()
   })
 
+  it('renders correct links', () => {
+    setup()
+    render(<TrialEligibleBanner />, { wrapper })
+
+    const upgradeLink = screen.getByRole('link', { name: /upgrade/ })
+    expect(upgradeLink).toBeInTheDocument()
+
+    const manageMembersLink = screen.getByRole('link', {
+      name: /manage members/,
+    })
+    expect(manageMembersLink).toBeInTheDocument()
```

Comment:
Slight tweak here to make sure the `href` is actually the one we want can you add in these checks:

```suggestion
    const upgradeLink = screen.getByRole('link', { name: /upgrade/ })
    expect(upgradeLink).toBeInTheDocument()
    expect(upgradeLink).toHaveAttribute('href', '<insert expected link>')

    const manageMembersLink = screen.getByRole('link', {
      name: /manage members/,
    })
    expect(manageMembersLink).toBeInTheDocument()
    expect(manageMembersLink).toHaveAttribute('href', '<insert expected link>')
```